example/test/pdf_text_test.dart

```diff
@@ -0,0 +1,128 @@
+@Skip('To run these tests, make sure an emulator or a real device is connected, then \'cd example\' and : \'flutter test/pfd_text_test.dart\'')
```

Comment:
Are you sure that it can be launched with this command? Shouldn't "flutter" be followed by a command? I could not run the tests :( .
Great job by the way! Can't wait to see it running.

Reply:
Woopsie, my bad! Absolutely you can run the tests like that! I have forgotten to add the 'run', so it is
```flutter run test/pdf_text_test.dart``` invoked from the example directory

Reply:
Added tests for password protected pdfs. The iOS part does not work (and I haven't touched it)

Reply:
Awesome, I could run the tests! Weird situation on iOS, password protected files still work for me using the example app.

Reply:
@AlessioLuciani Though it should not be the case, but perhaps it is related to the way the pdf is created? I created the one the tests are run against by simply exporting it from a LibreOffice Write document on Mac. If you could create a password protected pdf (with the password = password), confirm it works in the example app when run on iOS, then just stick it in the test/resources next to the one I created, we could modify the tests to point to the one that we know works in the example app and possibly figure out what the problem is?

Reply:
@hicnar now I have had a chance to try the test on iOS too (previously I had issues with running the iOS emulator and I only could on Android). I removed the if condition that stops the test for encrypted documents on iOS and run the test with your document, it seems to work for me! It does not prompt any errors and says that all tests passed. So I guess this is good news. Maybe is it that the problem is only on your emulator?

Reply:
@AlessioLuciani That's really interesting! Think in this case I will try it on a couple of more emulators!

Reply:
@hicnar no, what I described happened on my iOS emulator.